meta-lmp-base/recipes-sota/ostree-kernel-initramfs/ostree-kernel-initramfs_%.bbappend

```diff
@@ -17,3 +17,5 @@ do_install:append() {
 
 INITRAMFS_RECOVERY_IMAGE ?= ""
 do_install[depends] += "virtual/kernel:do_deploy ${@['${INITRAMFS_RECOVERY_IMAGE}:do_image_complete', ''][d.getVar('INITRAMFS_RECOVERY_IMAGE') == '']}"
+
+INITRAMFS_FSTYPES:remove:tegra = "cpio.gz.cboot"
```

Comment:
The problem on moving to a specific recipe is that the general setting of INITRAMFS_FSTYPES via append on the bsp layer influences every initrd that gets generated by the build, and why we removed on lmp-machine-custom (via remove).

The issue:
```
meta-tegra/conf/machine/include/tegra234.inc:INITRAMFS_FSTYPES:append = "${@'' if d.getVar('INITRAMFS_IMAGE_BUNDLE') == '1' else ' cpio.gz.cboot cpio.gz.cboot.bup-payload'}"
meta-tegra/conf/machine/include/tegra194.inc:INITRAMFS_FSTYPES:append = "${@'' if d.getVar('INITRAMFS_IMAGE_BUNDLE') == '1' else ' cpio.gz.cboot cpio.gz.cboot.bup-payload'}"
```

This forces this variable to be expanded on every recipe using it.

Reply:
yup, and maybe the `ostree-kernel-initramfs` is the reason we need to remove this types on lmp-machine-custom as this recipe refuses to build with it.

Reply:
Right, the reason is that in LmP we don't want to build any special type of initrd for the normal rootfs, since we're booting either via UEFI or u-boot, and on both the normal type is already enough.